security/csrf_in_login_form.rst

```diff
@@ -16,8 +16,13 @@ for CSRF. In this article you'll learn how you can use it in your login form.
 Configuring CSRF Protection
 ---------------------------
 
-First, make sure that the CSRF protection is enabled in the main configuration
-file:
+First, install the Security component in your project:
+
+.. code-block:: terminal
+
+    $ composer require security
```

Comment:
`security-csrf` is enough

Reply:
However, you need the Form component for the Twig function to be available.

Reply:
So, it should be `composer require security-csrf form`, correct?